server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -720,6 +720,12 @@ public class ShuffleServerConf extends RssBaseConf {
           .defaultValue(false)
           .withDescription("Whether to enable app detail log");
 
+  public static final ConfigOption<Boolean> SERVER_BITMAP_NEW_STRATEGY_ENABLED =
```

Comment:
Maybe a boolean type is not good for extension, it has only two state `true` or `false`, suggest another choice here  is that define a policy interface and two class implementation.
- CombinedBitmapPolicy
- PartitionBitmapPolicy

Than you can set it default to `CombinedBitmapPolicy` to keep consistent with original behavior.